### DISCUSSION TITLE
Prime react Tree : Need  "no results found"  message if no matches found in the filter

### DISCUSSION THREAD

**Discussion by ChithiraMG** [primefaces/community#605] [PrimeReact]
If there is no results matching the input filter value in tree component, it will be good if we show "no results found" message.
https://primereact.org/tree/#filter

**Comment by mertsincan** [comment]
Sorry for the delayed reponse! Thanks a lot for your feedback! @gucal @nitrogenous could you please create an issue and add this feature to the next major release?

**Comment by nitrogenous** [comment]
Thank you for your reviews! I have just created an issue for your need and We will be get it done in the next release! 

https://github.com/primefaces/primereact/issues/5813
